src/appmixer/google/gmail/ListThreads/ListThreads.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const GoogleApi = require('googleapis');
+const commons = require('../../google-commons');
+const { promisify } = require('util');
+
+// GoogleApi initialization & promisify of some api function for convenience
+const gmail = GoogleApi.gmail('v1');
+
+module.exports = {
+    async receive(context) {
+
+        const listThread = promisify(gmail.users.threads.list.bind(gmail.users.threads));
+        const { threads } = await listThread({
```

Comment:
why the google library? I think we'd like to use the "fetchData" in the new files, the same as in the ListSignatures.js